main/ertp/guide/amount-math.md

```diff
@@ -10,111 +10,88 @@ uses `amountMath` methods for all these operations.
 `amountMath` methods also check their arguments' `brand`, 
 throwing an error if the wrong `brand` was used.
 
-An `amountMath` is one of three different kinds, each of which
+An `amountMath` is one of two kinds, each of which
 implements the same methods. Which kind is used for a particular `brand` depends
 on what was specified when the `brand` and its `issuer` were 
 created. The kinds are: 
 - `MathKind.NAT` (`nat`): Used with fungible assets. `amount` `values` are natural numbers (non-negative BigInts).
-- `MathKind.STRING_SET` (`strSet`): Used with non-fungible assets. `amount` `values` are strings.
 - `MathKind.SET` (`set`): Used with non-fungible assets. `amount` `values` are objects or records with multiple properties.
```

Comment:
Let's fix the description of the values here to include arrays with any number of strings, numbers, etc.